defaults/settings.yml.default

```diff
@@ -82,6 +82,8 @@ sarotate:
   apprise:    #optional: apprise notifications (Default is blank)
 tandoor:
   secret_key: #Required: You can generate one with 'base64 /dev/urandom | head -c50'
+tqm:
+  download_client: #Required: Config default is qbt or deluge. 
```

Comment:
Need to rebase to pickup the changes to this file on master - also need to remove the comment as the changes reflect moving comments out of this file.

Reply:
I'll try to figure that out 😅 Will remove comments.